src/app/counter/counter-create-traveler/counter-create-traveler.component.ts

```diff
@@ -53,7 +56,10 @@ export class CounterCreateTravelerComponent implements OnInit {
     this.httpService
       .post(environment.counterUrl + environment.counterCreateUserUri, traveler)
       .subscribe(
-        (result) => this.dataService.setTraveler(result.body),
+        (result) => {
+          this.dataService.setTraveler(result.body);
```

Comment:
I commented on Sean's that it would be worthwhile to make these traveler and user data services into one service for all three users to ensure we don't have errant logins or data.